CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Authentication - Users can connect to their project using Google Single Sign-On.
+
+### Fixed
+- CORS - Update the CORS module to be compatible with Laravel >=5.5.0.
```

Comment:
You already did a PR for that.

Reply:
Yep, so it will not be a diff anymore when the cors pr will be merged and rebased here

Reply:
Did you test the compatibility with older projects?